src/addresses/mainnet.addresses.json

```diff
@@ -43,6 +46,9 @@
         "0x449D7C2e096E9f867339078535b15440d42F78E8",
         "0xA47D1251CF21AD42685Cc6B8B3a186a73Dbd06cf",
         "0xB70c25D96EF260eA07F650037Bf68F5d6583885e",
+        "0x2c72692E94E757679289aC85d3556b2c0f717E0E",
+        "0xEdf085f65b4F6c155e13155502Ef925c9a756003",
+        "0x415747EE98D482e6dD9B431fa76Ad5553744F247",
```

Comment:
This worries me, why did 2 new mainnet terms + pools just appear?

Reply:
I would imagine these could be dud deployments possibly? I would have to verify with Sarah but if it is the case I imagine we would have to filter these out

Reply:
I think some are related to the new DAI term

Reply:
https://github.com/element-fi/elf-deploy/commit/4d8ece7635e64136215d32b97d80c00fc32b966a#diff-0c3852a175371b4d102e1f7690cf6855ce0864c37d4ae89f697ee147cb2dcfc9R49-R51 - DAI Deployment